Sources/ArcGISToolkit/Components/FloorFilter/SiteAndFacilitySelector.swift

```diff
@@ -29,7 +29,30 @@ struct SiteAndFacilitySelector: View {
     private var isHidden: Binding<Bool>
     
     var body: some View {
-        SitesList(isHidden: isHidden)
+        NavigationView {
+            Group {
+                // If there's more than one site
+                if viewModel.sites.count > 1 {
+                    // Show the list of sites for site selection
+                    SitesList(isHidden: isHidden)
+                } else {
+                    // Otherwise there're no sites or only one site, show the list of facilities
+                    FacilitiesList(
+                        usesAllSitesStyling: false,
```

Comment:
Design question - Should the same change be made for facilities too i.e. if there is just one facility, skip showing the facility picker and directly show levels picker?

Reply:
> Design question - Should the same change be made for facilities too i.e. if there is just one facility, skip showing the facility picker and directly show levels picker? 

Good question! The design guideline (which all of the toolkit platforms adhere to) says:
> If there is only 1 facility, select it but still show the facilities browser.

[Source](https://devtopia.esri.com/runtime/common-toolkit/blob/main/designs/FloorFilter/FloorFilter.md#:~:text=If%20there%20is%20only%201%20facility%2C%20select%20it%20but%20still%20show%20the%20facilities%20browser.)

That being said, if you'd like, I'd be happy to provide this feedback to the toolkit team for consideration.

> If there is just one facility and one level, instead of showing any pickers, just shows the level directly on floor picker.

I believe this too would fall under the design guideline mentioned above.

> This also applies incase if there is not facility and just one level in the map.

I believe the design also covers this scenario.

> It is however not possible to have a Level without a Facility.

[Source](https://devtopia.esri.com/runtime/common-toolkit/blob/main/designs/FloorFilter/FloorFilter.md#:~:text=It%20is%20however%20not%20possible%20to%20have%20a%20Level%20without%20a%20Facility.)

Reply:
Yes, please get a feedback from the toolkit team. It's going to be less taps/selection for a user.

Reply:
Toolkit common # 208